pkg/server/glauth/handler.go

```diff
@@ -330,9 +331,9 @@ func parseFilter(f *ber.Packet) (qtype queryType, q string, err error) {
 		case "displayname":
 			q = fmt.Sprintf("display_name eq '%s'", escapeValue(value))
 		case "uidnumber":
-			q = fmt.Sprintf("uid_number eq '%s'", escapeValue(value))
+			q = fmt.Sprintf("uid_number eq %s", value) // TODO check it is a number?
```

Comment:
That todo can potentially come back in the future and bite us. Why not addressing it here? [There's this](https://golang.org/pkg/strconv/#ParseInt)